src/core/gltf2importer/gltf2importer.pri

```diff
@@ -43,7 +43,8 @@ SOURCES += \
     $$PWD/animationparser.cpp \
     $$PWD/sceneparser.cpp \
     $$PWD/materialparser.cpp \
-    $$PWD/skinparser.cpp
+    $$PWD/skinparser.cpp \
+    $$PWD/mikktspace/mikktspace.c
```

Comment:
Could it go in a 3rdparty folder inside of src?

Reply:
Im not sure how to do this. Do I have to add a new .pri file?

Reply:
Probably best... but maybe a bit involved for just a file. Or you could use relative paths too. Might have to do something regarding the include path in any cases.

Reply:
Done. Thanks Paul.